Authentication/SignUp/View/SignupViewConfiguration.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  SignupViewConfiguration.swift
+//  Authentication
+//
+//  Created by Daniela Riesgo on 4/1/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+
+public protocol SignupViewConfigurationType {
+    
+    var colourPalette: ColorPaletteType { get }
+    var fontPalette: FontPaletteType { get }
+    
+}
+
+public final class DefaultSignupViewConfiguration: SignupViewConfigurationType {
+    
+    public let colourPalette: ColorPaletteType
+    public let fontPalette: FontPaletteType
+    
+    public init() {
+        colourPalette = DefaultColorPalette()
+        fontPalette = DefaultFontPalette()
```

Comment:
I think the default configuration must have the default palette's because if not it's not a default one. If not we could change its name maybe for `SignupViewConfiguration` and make it a struct with contains the palettes received.